AstSemantics.md

```diff
@@ -220,12 +172,9 @@ MVP, there are [future features](FutureFeatures.md#finer-grained-control-over-me
 proposed to allow setting protection and creating mappings within the
 contiguous linear memory.
 
-The result type of `page_size` is `int32` for wasm32 and `int64` for wasm64.
-The result value of `page_size` is an unsigned integer which is a power of 2.
+The result type of `page_size` is `i32` for wasm32 and `i64` for wasm64.  The
+result value of `page_size` is an unsigned integer which is a power of 2.
 
-(Note that the `page_size` value need not reflect the actual internal page size
-of the implementation; it just needs to be a value suitable for use with
-`resize_memory`)
```

Comment:
This clarification is also guidance, much like footnotes are used in the C++ standard. The markdown formatting we're using here doesn't support footnotes, but this content feels to me like it belongs in this document regardless.